src/yaml/kellydale2003/31873-eighth-avenue-place-east-tower.yaml

```diff
@@ -0,0 +1,38 @@
+group: kellydale2003
+name: eighth-avenue-place-east-tower
+version: "1.0"
+subfolder: 300-commercial
+info:
+  summary: Eighth Avenue Place - East Tower
+  description: |-
+    **WARNING:** the immortal lot problem was detected in this model. This does not prevent it from being used in the game, only when demolished it's lot can remain. If this happens, use the [Immortality Killer](https://www.sc4devotion.com/csxlex/lex_filedesc.php?lotGET=2356) to get rid of the lot.
+
+    Eighth Avenue Place East Tower from Calgary, Canada.
+
+    3x5, 5x3 ploppable and growable.
+
+    **This was originally just a test export. But since the 3dsmax files have become corrupt, I can no longer finish the model. So, you'll notice details about the building a different and a few incomplete things here and there. Just thought I'd upload it as is rather than it go to the graveyard.**
+
+    **Oh well!**
+
+    No dependencies.
+
+    Enjoy!
+  author: kellydale2003
+  website: https://community.simtropolis.com/files/file/31873-eighth-avenue-place-east-tower/
+  images:
+    - https://www.simtropolis.com/objects/screens/monthly_2017_07/5593827217_d51e426a0a_b.jpg.40fedd9dde9eb1478eca947ce3cc165a.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2017_07/59753d02599f8_scr1316650974(2).jpg.c389178288d42403b204374db3f6c122.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2017_07/59753d0568230_St.Paul-Oct.51521500854094.png.606c78c083afc75a5468e979876dbf99.png
+    - https://www.simtropolis.com/objects/screens/monthly_2017_07/59753d994569e_St.Paul-Aug.181521500853851.jpg.8d619c72b2228e3aa983f78fa8c99146.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2017_07/59753da0434c9_St.Paul-Oct.231521500854177.png.47618fb8abca39e70c35d149a6507ae7.png
+    - https://www.simtropolis.com/objects/screens/monthly_2017_07/59753da3a8cf3_St.Paul-Sep.261521500854061.jpg.82899a6f75d6cc4acc0c6dc92607f8db.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2017_07/59753debd548a_St.Paul-Oct.141521500854135.jpg.4c15297f206bc7982ed98aebdec05738.jpg
+assets:
+  - assetId: kellydale2003-eighth-avenue-place-east-tower
```

Comment:
Although not explicitly mentioned in the description, this seems to be DN only, right?